### PR TITLE
Add script to automatize github release

### PR DESCRIPTION
**Pull request by qinqon**
It will instruct github to use GITHUB_USER and GITHUB_TOKEN as
user password and call `make release`

Signed-off-by: Quique Llorente <ellorent@redhat.com>

**What this PR does / why we need it**:

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by qinqon** [comment]
/hold

It's broken

File: automation/release.sh [hunk 1]
```diff
@@ -0,0 +1,17 @@
+#!/bin/bash -xe
+
+# This script tags current version and create a new release at github and pushes
+# to quay.io
+# To run it just do proper docker login and pass the git hub user password as
+# GIT_USER and GIT_PASSWORD env variables.
+
+# To pass user/password from automations, idea is taken from [1]
+# [1] https://stackoverflow.com/questions/8536732/can-i-hold-git-credentials-in-environment-variables
```
Comment (oshoval) [inline automation/release.sh]: nice
looking on
https://stackoverflow.com/questions/8536732/can-i-hold-git-credentials-in-environment-variables

isnt the last idea there is more stable ? (without sleep and race cond)

**Comment by qinqon** [comment]
/hold cancel
Now it's working

File: automation/release.sh (hunk 1 above)
Comment (qinqon) [reply automation/release.sh]: Didn't want to add another script just to print the password

File: automation/release.sh [hunk 2]
```diff
@@ -0,0 +1,17 @@
+#!/bin/bash -xe
+
+# This script tags current version and create a new release at github and pushes
+# to quay.io
+# To run it just do proper docker login and pass the git hub user password as
+# GIT_USER and GIT_PASSWORD env variables.
```
Comment (phoracek) [inline automation/release.sh]: You mention GIT_USER and GIT_PASSWORD, but the function uses GITHUB_USER and GITHUB_TOKEN

**Pull request merged**